rpi3/firmware/rpi3-bplus/config.txt

```diff
@@ -0,0 +1,3 @@
+enable_uart=1
```

Comment:
This belongs here, only thing is to change it to my proposed platform name instead `rpi-3bplus`.